### PR TITLE
Bugfix #446245: Conditions with sections

### PR DESCRIPTION
**Pull request by alexluckett**

### CODE REVIEW

**Comment by alexluckett** [comment]
With Jignesh for review.

File: src/server/plugins/engine/models/SummaryViewModel.ts
Comment (colinrotherham) [inline src/server/plugins/engine/models/SummaryViewModel.ts:174]: 🎉

File: src/server/plugins/engine/models/FormModel.ts
Comment (colinrotherham) [inline src/server/plugins/engine/models/FormModel.ts:154]: Can we put the ~`const`~ `let` back above and re-assign the schema every time it changes?

Think we should have code coverage for this (because everything still passed) 😱

```suggestion
      schema = schema.concat(page.stateSchema)
```

Comment (alexluckett) [reply src/server/plugins/engine/models/FormModel.ts:154]: Ahh!! Good spot, thank you.

**Review by colinrotherham — APPROVED** [review]
Looks loads better 🎉

Would be nice to start adding model tests but not a blocker

**Comment by alexluckett** [comment]
Waiting for Dave to merge #443 before merging.

**Review by davidjamesstone — APPROVED** [review]

**Pull request merged**
